packages/styling/src/styles/theme.ts

```diff
@@ -133,13 +135,21 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
 
   if (theme.defaultFontStyle) {
     for (const fontStyle of Object.keys(defaultFontStyles)) {
-      defaultFontStyles[fontStyle] = merge({}, defaultFontStyles[fontStyle], theme.defaultFontStyle);
+      // tslint:disable-next-line:no-any
+      (defaultFontStyles as any)[fontStyle] = merge({}, (defaultFontStyles as any)[fontStyle], theme.defaultFontStyle);
     }
   }
 
   if (theme.fonts) {
     for (const fontStyle of Object.keys(theme.fonts)) {
-      defaultFontStyles[fontStyle] = merge({}, defaultFontStyles[fontStyle], theme.fonts[fontStyle]);
+      // tslint:disable-next-line:no-any
```

Comment:
Not super important but if there's a series of lines with any I tend to do `// tslint:disable:no-any` at the beginning of the second and `// tslint:enable:no-any` after.

Reply:
my logic would be: lean towards single line disables unless there's a solid reason. it's easy to forget disabling, it's a little more work for a reader as it causes them to scan for the disable to make sure it wasn't forgotten. Plus writing it redundantly seems like a reasonable penalty as we should be avoiding them anyways.